samples/subsys/usb/dfu/sample.yaml

```diff
@@ -20,6 +20,7 @@ common:
     - mimxrt1180_evk/mimxrt1189/cm7
     - mimxrt700_evk/mimxrt798s/cm33_cpu0
     - nucleo_f207zg
+    - nucleo_h7s3l8
```

Comment:
Why is that necessary? I do no see test feature `usb_device` in https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/st/nucleo_h7s3l8/nucleo_h7s3l8.yaml

Reply:
correct, no need to exclude the nucleo_h7s3l8 at this time